### DISCUSSION TITLE
Roadmap of features

### DISCUSSION THREAD

**Discussion by enboig** [firefly-iii/firefly-iii#7404] [General]
In documentation it says it don't support sharing administration, but it is working on a better way to handle it.

Is there somewhere a roadmap of upcoming features (and an _estimated_ time...)?

**Comment by JC5** [comment]
There is, yes: https://roadmap.firefly-iii.org/

No timelines tho. It happens when it happens.

**Reply by enboig** [reply]
I saw the page, but missed the top tabs :sweat: 

Thanks
